### PR TITLE
docs(changelog): minor v10 edits

### PR DESCRIPTION
**Pull request by paulirish**
i was too slow. 

nothing massive in here. the changlog is a beaut.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -28,47 +28,41 @@ Thanks to our new contributors 👽🐷🐰🐯🐻!
...
 
-## New Audits
+## 🆕 New Audits
 
 ### Back/forward cache
 
-* bf-cache ([#14465](https://github.com/GoogleChrome/lighthouse/pull/14465))
-
-The Back/forward cache (bfcache for short) is a browser optimization that serves pages from fully serialized snapshots when navigating back or forwards in session history. There are over 100 different reasons why a page may not be eligible for this optimization, so to assist developers Lighthouse now attempts to trigger a bfcache response and will list anything that prevented the browser from using the bfcache.
+The Back/forward cache (bfcache for short) is a browser optimization that serves pages from fully serialized snapshots when navigating back or forwards in session history. There are over 100 different reasons why a page may not be eligible for this optimization, so to assist developers Lighthouse now attempts to trigger a bfcache response and will list anything that prevented the browser from using the bfcache. [#14465](https://github.com/GoogleChrome/lighthouse/pull/14465)
```
Comment (adamraine) [inline changelog.md:49]:
```suggestion
The back/forward cache (bfcache for short) is a browser optimization that serves pages from fully serialized snapshots when navigating back or forwards in session history. There are over 100 different reasons why a page may not be eligible for this optimization, so to assist developers Lighthouse now attempts to trigger a bfcache response and will list anything that prevented the browser from using the bfcache. [#14465](https://github.com/GoogleChrome/lighthouse/pull/14465)
```

**Review by adamraine — APPROVED** [review]

File: changelog.md (hunk 1 above)
Comment (connorjclark) [reply changelog.md:49]: i got this from the web dev article

Comment (connorjclark) [reply changelog.md:49]: which itself was inconsistent lol

File: changelog.md [hunk 2]
```diff
@@ -28,47 +28,41 @@ Thanks to our new contributors 👽🐷🐰🐯🐻!
...
+Our documentation is no longer hosted on web.dev. For the most up to date audit docs, please go to [developer.chrome.com/docs/lighthouse/](https://developer.chrome.com/docs/lighthouse/)
 
-## Breaking changes
-
-### For programmatic users
+## 💥 Breaking changes
 
-Under the hood, Lighthouse now uses the new user-flow supporting infrastructure by default, even for traditional navigation runs. You can opt out of this by: in the CLI, use `--legacy-navigation`; in DevTools: check “Legacy Navigation” in the settings menu. If you have a use case that necessitates this escape hatch, please file an issue. We plan to remove the legacy runner for 11.0.
+Under the hood, Lighthouse now uses the new user-flow supporting infrastructure by default, even for traditional navigation runs. You can opt out of this by: in the CLI, use `--legacy-navigation`; in DevTools: check “Legacy Navigation” in the settings menu. If you have a use case that necessitates this escape hatch, please file an issue. We plan to remove this legacy path in 11.0.
```
Comment (connorjclark) [inline changelog.md:65]: bad quotes

Comment (connorjclark) [reply changelog.md:65]: my fault ,sorry

**Review by connorjclark — APPROVED** [review]

**Comment by connorjclark** [comment]
LGTM, make sure to upstream to the release page when done.

File: changelog.md (hunk 1 above)
Comment (adamraine) [reply changelog.md:49]: Works for me

**Pull request merged**